### ISSUE TITLE
Add / Remove video during call

### ISSUE THREAD

**Issue by sfakir** [twilio/twilio-video.js#41]
* How is it possible to dettach the local video during the call? 
* How is it possible to limit the framerate to reduce CPU processing needs?

Thank you

**Comment by enricribas** [comment]
I would also like to get this working? 
Bueller?

**Comment by markandrus** [comment]
Hi @sfakir,

> How is it possible to dettach the local video during the call?

Are you asking about how to stop displaying the local video? If so, if you've started displaying local video using `LocalMedia#attach`, then you can simply call [`LocalMedia#detach`](http://media.twiliocdn.com/sdk/js/video/releases/1.0.0-beta3/docs/LocalMedia.html#detach__anchor); otherwise, if you've started displaying local video using `LocalVideoTrack#attach`, then you can simply call [`LocalVideoTrack#detach`](http://media.twiliocdn.com/sdk/js/video/releases/1.0.0-beta3/docs/LocalVideoTrack.html#detach__anchor). For examples,

```js
const localMedia = new Twilio.Video.LocalMedia();
// Add a LocalVideoTracks...
localMedia.attach('#local-media');

// Then, after some time...
localMedia.detach();
```

```js
const localVideoTrack = Array.from(localMedia.videoTracks.values())[0];
localVideoTrack.attach('#local-video-track');

// Then, after some time...
localVideoTrack.detach();
```

The [`detach` methods](http://media.twiliocdn.com/sdk/js/video/releases/1.0.0-beta3/docs/Media.html#detach) on Media, LocalMedia, and the various Track classes are similar in that they can take

* No arguments, i.e. "detach from _all_ elements";
* A query selector argument (shown above), i.e. "detach from any matching elements"; or,
* An HTMLElement argument, i.e. "detach from _this_ element".

@sfakir @enricribas Does this help? Regarding your other question,

> How is it possible to limit the framerate to reduce CPU processing needs?

We do not provide an API for this at this time; however, we are planning to address this next!

Thanks,
Mark

**Comment by mantra-valuecoders** [comment]
new Twilio.Video.LocalMedia(); is not available in updated version . please update answer

**Comment by sfakir** [comment]
The idea was to reduce bandwith in case of bad quality (similar to skype). Is this possible?

**Comment by markandrus** [comment]
We have a new set of APIs, `maxAudioBitrate` and `maxVideoBitrate`, in [ConnectOptions](http://media.twiliocdn.com/sdk/js/video/releases/1.6.0/docs/global.html#ConnectOptions) that let you reduce bandwidth. You can also update these while connected to a Room by calling [`setParameters`](http://media.twiliocdn.com/sdk/js/video/releases/1.6.0/docs/LocalParticipant.html#setParameters__anchor).

You can also pass the `frameRate` [MediaTrackConstraints](https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints) anywhere we accept it, e.g.

* `connect({ video: { frameRate: 15 } })`
* `createLocalTracks({ video: { frameRate: 15 } })`
* `createLocalVideoTrack({ frameRate: 15 })`
